.github/workflows/continuous-integration.yml

```
@@ -68,6 +68,8 @@ jobs:
               with:
                   token: ${{ secrets.CODECOV_TOKEN }}
                   file: ./coverage.xml
+            - name: fix code coverage path working-directory in coverage.xml
+              run: sed -i 's/\/home\/runner\/work\/kirin\/kirin\//\/github\/workspace\//g' coverage.xml
```

Other question (so different thread) : if it's working, using `$GITHUB_WORKSPACE` would be cleaner (like suggested in link from comment).
```suggestion
              run: sed -i 's#$GITHUB_WORKSPACE#/github/workspace/#g' coverage.xml
```

---

This did not work with the `$GITHUB_WORKSPACE` variable during my `sed` tests. So I leave the full path. Is this OK?